react/client/src/styles/breakpoints.js

```diff
@@ -1,7 +1,9 @@
 const device = {
-  sm: `(min-width: 320px)`,
-  // md: `(min-width: 768px)`,
-  md: `(min-width: 960px)`,
-  lg: `(min-width: 1200px)`
+  sm: 'min-width: 320px',
+  // md: '(min-width: 768px)',
+  md: '(min-width: 960px)',
+  lg: '(min-width: 1200px)',
+  sm_only: '@media (max-width: 320px)',
+  md_only: '@media (min-width: 321px) and (max-width: 960px)',
```

Comment:
I kind of want to replace the content of `sm` with what I put in `sm_only`, thoughts?

Reply:
Yep, makes sense to me